### PR TITLE
change symbol for railway=subway_entrance

### LINKED ISSUE #851: Change icon for railway=subway_entrance
Subway entrances are currently rendered with an icon that seems to symbolize 'walking'. This needs to be changed.

[Example](http://www.openstreetmap.org/#map=19/40.41912/-3.69332)

See also https://trac.openstreetmap.org/ticket/3920.

### PR DESCRIPTION
**Pull request by nebulon42**
Fixes #851, implements part of #1165.

before (click on images to see full preview)
subway_entrance_before

after
subway_entrance

issues:
~~subway entrances that are elevators are depicted with stairs (see http://www.openstreetmap.org/node/2703842678)~~

### CODE REVIEW

**Comment by matkoniecz** [comment]
Neither old nor new symbol works for me. Old one is clearly somebody walking what is not helpful and not representing anything. New one is not looking like stairs (also at full sized image) - it is rectangle with line.

Maybe just stairs would work? Or arrow pointed toward bottom (I am not really convinced that this is a good idea)?

**Comment by daganzdaanda** [comment]
Could you show the new symbol in train-blue? 
I think the idea works, if it is connected visually to other public transport features.
Or maybe it needs a filled rectangle to be more visible, or a tunnel-shape instead of a rectangle?

> Maybe just stairs would work? 

I've seen street signs where "stairs" stand for a pedestrian underpass. We probably don't need an icon for these, since we are rendering stairs and tunnels. So maybe it could work for subway entrance, too.

**Comment by nebulon42** [comment]
@daganzdaanda See https://github.com/gravitystorm/openstreetmap-carto/issues/851#issuecomment-98685860.

**Comment by nebulon42** [comment]
@matkoniecz Right, the stairs are barely visible, but visible. I still think the proposed icon is an improvement. Other opinions/ideas?

File: stations.mss [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 .stations {
   [railway = 'subway_entrance'][zoom >= 18] {
-    point-file: url('symbols/walking.n.12.png');
-    point-placement: interior;
+    marker-file: url('symbols/subway_entrance.10.svg');
+    marker-placement: interior;
+    marker-fill: #555;
```
Comment (pnorman) [inline stations.mss]: this should probably be transit blue

**Comment by kocio-pl** [comment]
Could we see how it looks now in blue on the map?

**Comment by nebulon42** [comment]
Some color variations:

grey
subway_entrance_after

station color
subway_entrance_station_color

station text color
subway_entrance_station_text_color

transportation color
subway_entrance_transport_color

**Comment by kocio-pl** [comment]
I see your examples in this order:
1. transportation (lighter than current, but still visible enough for this object - good for me)
2. grey
3. station
4. station text (map looks too much like a blue-red tonal rainbow =} )

**Comment by nebulon42** [comment]
Any other opinions on different colours?

**Comment by pnorman** [comment]
transportation colour.

**Comment by nebulon42** [comment]
Alternatively we could use a generic entrance icon:
subway_entrance

**Comment by kocio-pl** [comment]
Nice idea.

**Comment by nebulon42** [comment]
I presented two options that would work more or less and are clearly improvements over the current icon. A decision on this would be great.

**Comment by math1985** [comment]
I will get back to this as soon as possible.

**Comment by kocio-pl** [comment]
I think general entrance version is even better, since it's more clean and readable (which is important given it's smaller than most icons) and if we decide to render plain building entrances, they'll be probably black, so the distinction will be clear.

BTW: I think we can make [elevator icon](https://github.com/gravitystorm/openstreetmap-carto/issues/720) to be in the same style, just with arrows up and down.

**Comment by daganzdaanda** [comment]
Yeah, the general entrance with arrow is the better idea. The stairs are hard to decipher at that size and with the busy backgrounds. You probably already tried this, but maybe the arrow could point down 45° to symbolize entrance to the underground?

**Comment by kocio-pl** [comment]
Nice idea.

**Comment by matkoniecz** [comment]
Generic entrance works better for me.

**Comment by nebulon42** [comment]
The consensus seems to be the generic entrance icon. Updated.
For other PRs that might want to use a generic entrance icon, it is now available.

**Pull request merged**